### PR TITLE
feat(proposal-update): add ability to update proposal

### PR DESCRIPTION
**Pull request by Todmy**

### CODE REVIEW

File: src/sign/types.ts [hunk 1]
```diff
@@ -29,6 +29,19 @@ export interface Proposal {
...
+  proposal: string;
+  from: string;
+  space: string;
+  timestamp: number;
+  type?: ProposalType;
+  title?: string;
+  body?: string;
+  discussion?: string;
+  choices?: string[];
+  plugins?: string;
```
Comment (wa0x6e) [inline src/sign/types.ts:42]: Are those type really optional ? Isn't the signing failing if they're missing ?

Comment (ChaituVR) [reply src/sign/types.ts]: this is resolved?

Comment (wa0x6e) [reply src/sign/types.ts]: I believe so

**Review by wa0x6e — APPROVED** [review]
tAck

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@snapshot-labs/snapshot.js",
-  "version": "0.5.9",
+  "version": "0.5.10",
```
Comment (ChaituVR) [inline package.json:3]: Feel like this is a slightly bigger update :P
```suggestion
  "version": "0.6.0",
```

File: src/sign/types.ts [hunk 3]
```diff
@@ -126,6 +139,21 @@ export const proposalTypes = {
   ]
 };
 
+export const updateProposalTypes = {
+  updateProposal: [
```
Comment (ChaituVR) [inline src/sign/types.ts:143]: All other types start with a capital letter, maybe better to make it `UpdateProposal`?

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**